apollo-federation/src/query_plan/fetch_dependency_graph.rs

```diff
@@ -203,7 +204,7 @@ type FetchDependencyGraphPetgraph =
 pub(crate) struct FetchDependencyGraph {
     /// The supergraph schema that generated the federated query graph.
     #[serde(skip)]
-    supergraph_schema: ValidFederationSchema,
+    pub(crate) supergraph_schema: ValidFederationSchema,
```

Comment:
is this a leftover from testing or there is a reason for this to be `pub(crate)`?